siobrultech_protocols/gem/protocol.py

```diff
@@ -76,25 +76,33 @@ def _get_packet(self) -> Optional[Packet]:
 
             def skip_malformed_packet(msg: str, *args: Any, **kwargs: Any):
                 LOG.debug(
-                    "Skipping malformed packet due to " + msg + ". Buffer contents: %s",
+                    "%d Skipping malformed packet due to "
+                    + msg
+                    + ". Buffer contents: %s",
```

Comment:
Would it be better to make `msg` a `%s` in the string as well?

Reply:
We're passing through `*args` as well, which means `msg` is intended to be able to have format specifiers.